Jellyfin.Server.Implementations/MediaSegments/MediaSegmentManager.cs

```diff
@@ -103,4 +177,22 @@ public bool IsTypeSupported(BaseItem baseItem)
     {
         return baseItem.MediaType is Data.Enums.MediaType.Video or Data.Enums.MediaType.Audio;
     }
+
+    /// <inheritdoc/>
+    public IEnumerable<(string Name, string Id)> GetSupportedProviders(BaseItem item)
+    {
+        if (item is not (Video or Audio))
```

Comment:
MediaSegments should only be available for `Video`

Reply:
That would contradict our previous discussion
https://github.com/jellyfin/jellyfin/pull/12345#discussion_r1693925815